aspoet/src/main/kotlin/com/google/androidstudiopoet/models/AndroidBuildGradleBlueprint.kt

```diff
@@ -23,17 +23,17 @@ import com.google.androidstudiopoet.input.PluginConfig
 import com.google.androidstudiopoet.utils.joinPath
 
 class AndroidBuildGradleBlueprint(val isApplication: Boolean, private val enableKotlin: Boolean, val enableDataBinding: Boolean,
-                                  moduleRoot: String, private val androidBuildConfig: AndroidBuildConfig, val packageName: String,
-                                  val extraLines: List<String>?, productFlavorConfigs: List<FlavorConfig>?,
+                                  moduleRoot: String, androidBuildConfig: AndroidBuildConfig, val packageName: String,
+                                  override val extraLines: List<String>?, productFlavorConfigs: List<FlavorConfig>?,
                                   buildTypeConfigs: List<BuildTypeConfig>?, additionalDependencies: Set<Dependency>,
-                                  pluginConfigs: List<PluginConfig>?) {
-    val plugins: Set<String> = createSetOfPlugins(pluginConfigs)
+                                  pluginConfigs: List<PluginConfig>?) : AndroidModuleBuildSpecificationBlueprint {
+    override val plugins: Set<String> = createSetOfPlugins(pluginConfigs)
```

Comment:
I'm not sure if this will work on Windows... You can pass it in the constructor. It is called from `ModuleBlueprint`, so you can use `AbstractModuleBlueprint.name` property there. It has the value you are looking for..